Makefile

```diff
@@ -6,6 +6,16 @@ define eachmod
 	@find . -type f -name go.mod -print0 | xargs -I '{}' -n1 -0 bash -c 'dir="$$(dirname {})" && echo "$${dir}" && cd "$${dir}" && $(1)'
 endef
 
+all: clean docker-ipfs-testground install-testground
+
+clean:
+	docker rm -f testground-sidecar || true
+	docker rm -f testground-redis || true
+	docker rm -f testground-goproxy || true
+
+install-testground:
```

Comment:
A few comments:

* We can safely drop `-testground` suffix; it's redundant.

* Also, let's rename the `docker-ipfs-testground` target to `docker`.

* `clean` is a misnomer here; `make clean` is commonly used to clean build targets, not to delete docker containers. Let's look for another name.

Reply:
I was hesitant to change `docker-ipfs-testground` as it has been used for ages, but we are in v0.4 so probably best to introduce better naming now, than later.

Updated these, let me know if you think they are better.